Gems/EMotionFX/Code/Tests/RenderBackendManagerTests.cpp

```diff
@@ -158,7 +163,8 @@ namespace EMotionFX
                     m_actorInstance,
                     m_actorAsset,
                     SkinningMethod::Linear,
-                    /*transform=*/{}));
+                    /*transform=*/{},
+                    true));
```

Comment:
/* rayTracingEnabled= */ true

Reply:
This has been added as a part of the newest changes